Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8438,140 +8494,235 @@ def prepare_readable_ip_data(response):
 '''COMMANDS'''
 
 
-def domain_command(reliability):
```

Comment:
This function was removed because it's the same logic, input as `whois_command`. No need to maintain two identical functions.

Reply:
But it is not entirely correct as the domain logic should have the "generic command" [outputs](https://xsoar.pan.dev/docs/integrations/context-standards-mandatory#domain) whereas the whois command does not.

Reply:
Also It is usually recommended  to create such wrappers so you could easily extract different arguments for different functions.

Reply:
I see what you mean. In the case with the code in `domain_command` and `whois_command` currently available in `master`, what is returned is identical:

```python
# domain_command

demisto.results({
            'Type': entryTypes['note'],
            'ContentsFormat': formats['markdown'],
            'Contents': str(whois_result),
            'HumanReadable': tableToMarkdown('Whois results for {}'.format(domain), md),
            'EntryContext': dbot_score,
        })

# whois_command

demisto.results({
            'Type': entryTypes['note'],
            'ContentsFormat': formats['markdown'],
            'Contents': str(whois_result),
            'HumanReadable': tableToMarkdown('Whois results for {}'.format(domain), md),
            'EntryContext': context_res,
        })
```

Therefore, since there doesn't seem to be a wrapper in this case, I decided to remove one of them for easier maintainability.

Reply:
Let's do this, since merging both logics into one amends you need to take thee extraction of args outside from the whois_command (different from what happens today), let's add a wrapper command for each of the flows (whois_command and domain_command where each is making the needed preparations relevant foe its flow (extraction of args, initialization of variables, etc...)

Reply:
https://github.com/demisto/content/pull/27370/commits/5fd5e32059294aed675c92a01b757f3257911d77